### PR TITLE
arch: arm: kinetis: Author Gregory Nutt: update licenses to Apache

### PR DESCRIPTION
**Pull request by jerpelea**
## Summary
Gregory Nutt has submitted the SGA and we can migrate the licenses to Apache.

## Impact
NONE

## Testing
NONE

### CODE REVIEW

File: arch/arm/src/kinetis/kinetis_pwm.h [hunk 1]
```diff
@@ -79,13 +81,13 @@
 #include <arch/board/board.h>
```
Comment (davids5) [inline arch/arm/src/kinetis/kinetis_pwm.h:81]: Does this matter?

**Review by davids5 — APPROVED** [review]
Some n->N still left

**Comment by jerpelea** [comment]
@xiaoxiang781216 please ignore warning: #include outside of 'Included Files' section

**Review by xiaoxiang781216 — APPROVED** [review]
LGTM.

**Comment by jerpelea** [comment]
can we merge it?

**Pull request merged**